tests/test_theseus_layer.py

```diff
@@ -353,12 +353,13 @@ def cost_weight_fn():
             l2_reg = F.mse_loss(
                 cost_weight_fn(), torch.zeros((1, num_points), device=device)
             )
-            loss = (cost_gt - cost_opt) + 10.0 * l2_reg
+            loss = (cost_gt - cost_opt) ** 2 + 10.0 * l2_reg
             loss = torch.mean(loss, dim=0)
         else:
             loss = mse_loss
 
         loss.backward()
+        print("Loss: ", loss.item())
```

Comment:
Does this print need to be inside `if verbose:`?

Reply:
I think it's OK to leave as is, easier to see what's going on if it ever fails in CI. `verbose=True` was printing all the optimizer iterations which was way too much stuff.